src/Benchmark/StatSendingBenchmark.cs

```diff
@@ -28,11 +28,11 @@ public void Setup()
             var endpointSource = EndpointParser.MakeEndPointSource(
                 config.Host, config.Port, config.DnsLookupInterval);
 
-            var ipTransport = new IpTransport(endpointSource);
+            var ipTransport = new SocketTransport(endpointSource, SocketProtocol.Ip);
```

Comment:
I _think_ that naming guidelines is that for acronyms of two letters you capitalise, so `IP` rather than `Ip`.

Reply:
correct, "capitalizes the first character of each word (including acronyms over two letters in length),"

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions